MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -170,12 +170,12 @@ class RouteManeuverViewController: UIViewController {
     }
     
     func updateStreetNameForStep() {
-        if let step = step, step.isNumberedMotorway, let codes = step.codes {
+        if let destinations = step?.destinations {
+            destinationLabel.unabridgedText = destinations.joined(separator: NSLocalizedString("DESTINATION_DELIMITER", bundle: .mapboxNavigation, value: " / ", comment: "Delimiter between destinations in a road concurrency"))
```

Comment:
Here is how small it gets and how much we can stuff in there:

![simulator screen shot jul 25 2017 11 18 19 am](https://user-images.githubusercontent.com/1058624/28587174-17600c88-712b-11e7-8e89-05706a3dd60a.png)

I think this is okay.